### PR TITLE
DOC: fix SA01, ES01 for pandas.Series.sparse.npoints

### PR DESCRIPTION
**Pull request by tuhinsharma121**
fixes

```
pandas.Series.sparse.npoints SA01, ES01
```

### CODE REVIEW

File: pandas/core/arrays/sparse/array.py [hunk 1]
```diff
@@ -708,6 +708,18 @@ def npoints(self) -> int:
...
+        This property returns the number of elements in the sparse series that are
+        not equal to the ``fill_value``. Sparse data structures store only the
+        non-``fill_value`` elements, reducing memory usage when the majority of
+        values are the same.
+
+        See Also
+        --------
+        Series.sparse.to_dense : Convert a Series from sparse values to dense.
+        Series.sparse.fill_value : Elements in `data` that are `fill_value` are
+            not stored.
```
Comment (tuhinsharma121) [inline pandas/core/arrays/sparse/array.py:720]:
```suggestion
        Series.sparse.fill_value : Elements in ``data`` that are ``fill_value`` are
            not stored.
```

Comment (tuhinsharma121) [reply pandas/core/arrays/sparse/array.py:720]: @mroeschke do you think double tick is needed here? I want to understand where to use single tick vs double tick. Appreciate your explanation in advance!

Comment (mroeschke) [reply pandas/core/arrays/sparse/array.py:720]: Yeah I think double ticks would be good here. Generally I think arguments to methods/functions or any Python identifiers should use double ticks

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @tuhinsharma121

**Pull request merged**